shared/js/background/chrome-events.es6.js

```diff
@@ -486,7 +493,13 @@ chrome.webRequest.onBeforeSendHeaders.addListener(
             const tab = tabManager.get({ tabId: request.tabId })
             if (!requestHeaders) return
             if (tab && tab.site.whitelisted) return
-            if (tab && utils.isFirstParty(request.url, tab.url)) return
+            if (!tab && request.tabId === -1) {
+                if (utils.isFirstParty(request.initiator, request.url)) {
```

Comment:
Why not put this all into one if statement?

Reply:
Felt messier but I have no qualms about it